files/image_config/monit/control_plane_drop_check

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/env python3
+"""
+    This scripts's job is to verify that there are no control plane
+    packet drops reported by /proc/net/sofnet_stats.
+    This is to be run periodically on a SONiC device using a monit
+    configuration file.
+"""
+import os
+import sys
+import syslog
+
+
+def write_syslog(message, *args):
+    """
+    Write a message to syslog.
+
+    Args:
+        message (str): Message string to be logged
+        args: Optional args
+
+    Returns:
+        None
+    """
+
+    if args:
+        message %= args
+    syslog.syslog(syslog.LOG_NOTICE, message)
+
+
+def get_softnet_dropped_count():
+    """
+    Get dropped count from softnet stats procfs.
+
+    Returns:
+        drop_count (int): Number of dropped packets
+    """
+    drop_count = 0
+    softnet_stats_file = "/proc/net/softnet_stat"
+
+    with open(softnet_stats_file, 'r') as f:
+        for line in f:
+            if line.strip():  # Ensure the line is not empty
+                stat = line.split()
+                # Drop count is in the second column for each CPU
```

Comment:
I think second column in softnet_stat represents drop counter in all CPU architectures, please add a note.

Reply:
Okay, I'll confirm for all cpu architectures.

Reply:
Per https://github.com/torvalds/linux/blob/v5.10/net/core/net-procfs.c#L153, the dropped count seems to be always in column 2 independent of the cpu architecture.

Reply:
Please include this reference in the comment.

Reply:
Addressed.